exonum-node/src/events_impl.rs

```diff
@@ -115,29 +120,25 @@ impl NodeHandler {
         self.blockchain.shutdown();
     }
 
-    fn flush_txs_into_pool(&mut self) {
+    pub(crate) fn flush_txs_into_pool(&mut self) {
         let tx_cache_size = self.state().tx_cache_len();
-
         if tx_cache_size == 0 {
-            //No need to do anything.
-            trace!("Transaction cache is empty.");
             return;
+        } else {
+            info!(
+                "Flushing {} transactions from pool to storage",
```

Comment:
Like in place below `... from cache to persistent pool` is more appropriate, I suppose.